doc/source/contributing/index.rst

```diff
@@ -124,22 +124,25 @@ To run all the tests (except the virtual instruments)
 .. code-block:: sh
 
     cd scikit-rf
-    pytest
+    python -m venv .venv
+    python -e[test,visa,netw,xlsx,plot,docs,testspice]
 
-Or, to run tests for every installed Python installation in an isolated environment.
+    # On Linux Systems
+    . .venv/bin/activate
 
-.. code-block:: sh
+    # On Windows
+    .\.venv\Scripts\activate
 
-    tox
+    pytest
 
 To run all tests *and* all tutorials and example notebooks (recommended before making a pull request):
 
 .. code-block:: sh
 
-    tox -- --nbval-lax
+    pytest --nbval-lax
```

Comment:
does it requires the previous venv commands before it runs?

Reply:
At least the activation stuff.